R/PipeOpFlatFun.R

```diff
@@ -13,9 +13,10 @@
 #'   of features to keep. The flattening is only applied to those columns.\cr
 #'   See [`Selector`][mlr3pipelines::Selector] for example functions. Default is
 #'   selector_all()`, which selects all of the `functional` features.
-#' * `interpolate` :: `logical(1)` \cr
-#'   Should functions be evaluated (i.e., inter-/extrapolated) for arg for which no original data is available?
-#'   Default is `FALSE`.
+#' * `grid` :: `character(1)` | numeric() \cr
+#'   The grid to use for interpolation. If `grid` is a character, it must be either `"union"` or `"intersect"`.
+#'   If `grid` is numeric, it must be a sequence of values to use for the grid.
+#'   Default is `"union"`.
```

Comment:
default is not the same as initial value

Reply:
also explain what intersect and union do

Reply:
in general I think it would be good to describe better what this pipeop does exactly (regarding NAs etc)

Reply:
also we need to document how the values are interpolated (which function)